src/lib/converter/converter.ts

```diff
@@ -338,6 +338,19 @@ export class Converter extends ChildableComponent<Application, ConverterComponen
             result.notSupportedIn = tag[1].split(/,\s?/);
         }
 
+        if (result && comment != null && comment.indexOf('@examples') != -1) {
+            var examplesTagRegex = /@(?:examples)\s*((?:'[^'\\]*(?:\\.[^'\\]*)*',\s*)*'[^'\\]*(?:\\.[^'\\]*)*')/;
+
+            result.comment = parseComment(comment.replace(examplesTagRegex, ''));
+
+            var tag = examplesTagRegex.exec(comment);
+            if (!tag[1]) {
```

Comment:
It's possible the exec returns `null` if the match should fail.
Maybe `!tag || !tag[1]` would be safer

Reply:
Good idea, will do!